lib/improver/wind_downscaling.py

```diff
@@ -89,10 +89,10 @@ def __init__(self, u_href, h_ref, z_0, mask):
         self.z_0 = z_0
         self.mask = mask
 
-    def calc_ustar(self):
+    def process(self):
```

Comment:
It might also to good to add to the `__init__`, a check that the arrays are the same size. Currently, if the arrays are different sizes, the error returned will be `IndexError: boolean index did not match indexed array along dimension 0; dimension is 4 but corresponding boolean dimension is 2`, which isn't particularly clear, so it might be worth raising an error, if there is a size mismatch in the arrays, and adding a unit test to check that this exception is raised correctly.